runbot_send_email/data/runbot_send_email_data.xml

```diff
@@ -222,8 +181,8 @@ Build #${object.sequence} is ${object.state} but failed
           <td style="padding:0px;border:0px">
           </td>
           <td style="padding:0px;border:0px;color:#808080">
-              <a href="${object.doc_link}" style="text-decoration:underline;color:#606060" target="_blank">Documentation</a> about <span class="il">Vauxoo's Runbot</span> and <a href="${object.dockerdoc_link}" style="text-decoration:underline;color:#606060" target="_blank">generated docker</a>. <br>
-                  For help please send a message to Vauxoo's telegram group, choose who receive this notification in your <a href="${object.configfile_link}" style="text-decoration:underline;color:#606060" target="_blank">Config File</a>.
+              <a href="/runbot_doc/static/index.html" style="text-decoration:underline;color:#606060" target="_blank">Documentation</a> about <span class="il">Vauxoo's Runbot</span> and <a href="https://github.com/Vauxoo/travis2docker/wiki" style="text-decoration:underline;color:#606060" target="_blank">generated docker</a>. <br>
```

Comment:
where is this ur?

Reply:
@moylop260 the URL is complete with the host before it send the email.
![image](https://user-images.githubusercontent.com/17712814/38429452-028ab408-3984-11e8-8c81-bc1579df2c7c.png)